specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-10-01-preview/workspaceRP.json

```diff
@@ -6903,6 +7081,10 @@
             "type": "string"
           }
         },
+        "lazyInitSystemDatastores": {
```

Comment:
this is already integrated with our down stream services and we are pretty sure we only need true/false here, I will keep this for now, but thanks for the advice we will consider using enum instead of string in the future properties.